### PR TITLE
-n flag

### PR DESCRIPTION
**Pull request by Euro20179**
does the selection process n times

example:
`ytfzf -aL -n2 hello`
prints
```
https://www.youtube.com/watch?v=YQHsXMglC9A
https://www.youtube.com/watch?v=mHONNcZbwDY
```

### CODE REVIEW

**Comment by JojiiOfficial** [comment]
Doesn't -L already do this?

**Comment by Euro20179** [comment]
> Doesn't -L already do this?

No because if you do -L and don't want the menu screen you can't select more than 1 link

**Comment by Euro20179** [comment]
don't merge this yet because there some oversights.

**Comment by JojiiOfficial** [comment]
> No because if you do -L and don't want the menu screen you can't select more than 1 link
Good point. Didn't even notice the '-L' in your example query.

**Comment by pystardust** [comment]
Nice idea, but doest it suffice to change
```
if [ $auto_select -eq 1 ] ; then
		    selected_data="$selected_data\n$(echo "$videos_data" | sed ${i}q | tail -n1)" ;
```
This would select the first `i` videos

File: ytfzf [hunk 1]
```diff
@@ -275,24 +276,31 @@ scrape_fn () {
...
-		selected_data="$(echo "$videos_data" | shuf -n 1)"
+		selected_data="$(echo "$videos_data" | shuf -n $link_count )"
 	elif [ $show_thumbnails -eq 1 ] ; then
 		dep_ck "ueberzug"
 		start_ueberzug
 		selected_data="$(echo "$videos_data" | video_menu_img )"
 		stop_ueberzug 
 	else
-		selected_data="$(echo "$videos_data" | video_menu )"
+		selected_data="$selected_data\n$(echo "$videos_data" | video_menu )"
```
Comment (pystardust) [inline ytfzf:290]: we wouldn't need to append cause no data is being selected before this

**Pull request merged**